File: televisionbingecalculator/src/main/java/com/vanillax/televisionbingecalculator/app/ServerAPI/movie/Offer.java
package com.vanillax.televisionbingecalculator.app.ServerAPI.movie;

import com.google.gson.annotations.SerializedName;

/**
 * Created by mitchross on 5/5/17.
 */

public class Offer
{
	public static final int VUDU = 7;
	public static final int NETFLIX = 8;
	public static final int AMAZON = 10;
	public static final int HBO = 31;
	public static final int HULU=15;


	@SerializedName("provider_id")
	public int providerId;
}

File: televisionbingecalculator/src/main/java/com/vanillax/televisionbingecalculator/app/Kotlin/network/response/Offer.kt
package com.vanillax.televisionbingecalculator.app.Kotlin.network.response

import com.google.gson.annotations.SerializedName

/**
 * Created by mitchross on 5/5/17.
 */

class Offer {


    @SerializedName("provider_id")
    var providerId: Int = 0

    enum class Streamtype (val value: Int ) {
         VUDU(7),
         NETFLIX (8),
         AMAZON (10),
         HBO (31),
         HULU  (15)
    }


}
